python-stdlib/binascii/binascii.py

```diff
@@ -331,7 +331,7 @@ def a2b_base64(ascii):
 table_b2a_base64 = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/"
 
 
-def b2a_base64(bin):
+def b2a_base64(bin, newline=True):
```

Comment:
This change doesn't seem to have anything to do with urequests, but otherwise it's OK to have here (it's a separate commit, which is good).

Reply:
Yeah I'd thought it was related - was in the same original PR as the redirect/chunked change. Can confirm it was not actually used there, though looks like a clean change anyway.